pkgs/development/python-modules/earcut/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  yq-go,
+  lib,
+  numpy,
+  fetchPypi,
```

Comment:
AFAIK nixpkgs always prefer to build from source. It's better for traceability and security purpose (xyz kof kof). Can you use `fetchFromGitHub` (mind the capital H) instead?

Reply:
Do you confirm that this is the earcut GH repo ? https://github.com/mapbox/earcut

Reply:
latest version on the mapbox repo is 3.0.0. Should I add this rev or stick to the 1.1.5 specified on Pypi? Thanks @autra for your feedback :)

Reply:
No, the pypi earcut package is https://pypi.org/project/earcut/, which points to https://github.com/vojtatom/earcut.py

We could also package mapbox-earcut, but it's not useful for py3dtiles.